src/vivarium_public_health/risks/distributions.py

```diff
@@ -85,18 +85,18 @@ class RebinPolytomousDistribution(DichotomousDistribution):
     pass
 
 
-def get_distribution(risk: str, risk_type: str, builder):
-
-    distribution_type = builder.data.load(f"{risk_type}.{risk}.distribution")
-    exposure_data = builder.data.load(f"{risk_type}.{risk}.exposure")
+def get_distribution(risk: str, distribution_type: str, exposure_data: pd.DataFrame, **data):
 
     if distribution_type == "dichotomous":
         exposure_data = pivot_age_sex_year_binned(exposure_data, 'parameter', 'value')
         distribution = DichotomousDistribution(exposure_data, risk)
 
     elif distribution_type == 'polytomous':
+        if "configuration" not in data:
```

Comment:
I think we'd be okay dropping the error checking for the data dictionary.  This function is unlikely to ever be called except by the risk component and the default key error you'd get should be relatively straightforward to debug given the construction of the data dict is one frame up.